src/cppnv/EnvKey.h

```diff
@@ -0,0 +1,41 @@
+#ifndef SRC_CPPNV_ENVKEY_H_
+#define SRC_CPPNV_ENVKEY_H_
+#include <string>
+
+namespace cppnv {
```

Comment:
These should likely be under a top level `namespace node` for consistency with the rest of the codebase

Reply:
will do

Reply:
I thought about this, I haven't done it in the current push. Shouldn't we want to not pollute the node namespace with a utility library. If you say yes, I'll do it, however, something keeps ticking me when I go to.